src/navigation/baby_slam/baby_slam/map_cone.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+from turtle import color
+from typing import Tuple, Optional
+import math
+
+from driverless_msgs.msg import Cone
+
+@dataclass
+class MapCone:
+    """Creates a datatype for storing and converting cones recorded"""
+
+    def __init__(self, cone: Cone):
+        self.x = cone.location.x
+        self.y = cone.location.y
+        self.colour = cone.color
+
+        self.range = math.hypot(self.x, self.y)
+        self.bearing = math.atan2(self.y, self.x)
+
+    @property
+    def sense_rb(self) -> Tuple:
```

Comment:
put the types inside the tuple in square brackets after the Tuple type
e.g. ` def sense_rb(self) -> Tuple[float, float]:`